prestodb/src/main/java/io/pravega/connectors/presto/util/PravegaNameUtils.java

```diff
@@ -51,20 +52,24 @@ public static String groupId(String scope, String stream)
         return scope + "." + stream;
     }
 
-    // test stream name - if not valid pravega stream name assume it is regex for multi source
     public static boolean multiSourceStream(PravegaStreamDescription object)
     {
+        // if stream name is a regex, or if there are object args
+        // (objectArgs for stream are comma sep list of component streams)
         return object.getObjectType() == ObjectType.STREAM &&
-                multiSourceStream(object.getObjectName());
+                (multiSourceStream(object.getObjectName()) || object.getObjectArgs().isPresent());
```

Comment:
I'm not sure I understand this predicate, why is it correct to check for the presence of object args?

Reply:
for streams, you can explicitly list out what the component streams will be (vs. regex)
https://github.com/pravega/presto-connector/wiki/Schema-Registry#explicitly-listed

if object is STREAM, objectArgs can be this list

if object is KV_TABLE objectArgs will be list of key families